packages/builder/src/steps/router.ts

```diff
@@ -135,13 +140,16 @@ const routerStep = {
     const inputData = getCompileInput(contractName, sourceCode, evmVersion);
     const solidityInfo = await compileContract(contractName, sourceCode, evmVersion);
 
-    // the abi is entirely basedon the fallback call so we have to generate ABI here
+    // the ABI is entirely based on the fallback call so we have to generate ABI here
     const routableAbi = getMergedAbiFromContractPaths(ctx, config.contracts);
```

Comment:
technically this variable name is now inaccurate because the constructor is not "routable"